### PR TITLE
Verify SAML setup documentation for NameID

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/docs/issues/4267

Updated:
- Self Managed Admin Guide > Onboard Users > SAML Single Sign-On (E20) > Configure SAML with Microsoft ADFS for Windows Server 2012 > Create Claim Rules
   - Updated step 3 to add another bullet point for creating a claim rule for `NameID`

### CODE REVIEW

File: source/deployment/sso-saml-adfs-msws2016.rst [hunk 1]
```diff
@@ -83,12 +83,13 @@ Create Claim Rules
...
-  - From the LDAP Attribute column, **select** ``Surname``. From the Outgoing Claim Type, **type** ``LastName``
-  - From the LDAP Attribute column, **select** ``SAM-Account-Name``. From the Outgoing Claim Type, **type** ``Username``
+  - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``Email``.
+  - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``NameID``.
+  - From the **LDAP Attribute column**, select ``Given-Name``. From the **Outgoing Claim Type**, type ``FirstName``.
+  - From the **LDAP Attribute column**, select ``Surname``. From the **Outgoing Claim Type**, type ``LastName``.
+  - From the **LDAP Attribute column**, select ``SAM-Account-Name``. From the **Outgoing Claim Type**, type ``Username``.
 
-For Mattermost 3.4 and later, the *FirstName* and *LastName* attributes are optional.
+From Mattermost 3.4, the *FirstName* and *LastName* attributes are optional.
```
Comment (justinegeffen) [inline source/deployment/sso-saml-adfs-msws2016.rst:92]:
```suggestion
From Mattermost version 3.4, the ``FirstName`` and ``LastName`` attributes are optional.
```

File: source/deployment/sso-saml-adfs.rst [hunk 2]
```diff
@@ -85,11 +85,12 @@ Create Claim Rules
 
 3. In the **Configure Claim Rule** screen, enter a **Claim Rule Name** of your choice, select **Active Directory** as the **Attribute Store** and do the following:
   - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``Email``.
+  - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``NameID``.
   - From the **LDAP Attribute column**, select ``Given-Name``. From the **Outgoing Claim Type**, type ``FirstName``.
   - From the **LDAP Attribute column**, select ``Surname``. From the **Outgoing Claim Type**, type ``LastName``.
   - From the **LDAP Attribute column**, select ``SAM-Account-Name``. From the **Outgoing Claim Type**, type ``Username``.
 
-For Mattermost 3.4 and later, the *FirstName* and *LastName* attributes are optional.
+From Mattermost 3.4, the *FirstName* and *LastName* attributes are optional.
```
Comment (justinegeffen) [inline source/deployment/sso-saml-adfs.rst:93]:
```suggestion
From Mattermost version 3.4, the ``FirstName`` and ``LastName`` attributes are optional.
```

**Review by justinegeffen — COMMENTED** [review]
I changed the formatting of the attribute names to match the formatting in the list - italics seemed out of place. The other option is to make the format bold to match UI formatting convention.

File: source/deployment/sso-saml-adfs-msws2016.rst [hunk 3]
```diff
@@ -83,12 +83,13 @@ Create Claim Rules
 	.. image:: ../../source/images/SSO-SAML-ADFS_create-claim-rules_002.png
 
 3. In the **Configure Claim Rule** screen, enter a **Claim Rule Name** of your choice, select **Active Directory** as the **Attribute Store** and add the following mapping:
-  - From the LDAP Attribute column, **select** ``E-Mail-Addresses``. From the Outgoing Claim Type, **type** ``Email``
-  - From the LDAP Attribute column, **select** ``Given-Name``. From the Outgoing Claim Type, **type** ``FirstName``
-  - From the LDAP Attribute column, **select** ``Surname``. From the Outgoing Claim Type, **type** ``LastName``
-  - From the LDAP Attribute column, **select** ``SAM-Account-Name``. From the Outgoing Claim Type, **type** ``Username``
+  - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``Email``.
+  - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``NameID``.
```
Comment (sbishel) [inline source/deployment/sso-saml-adfs-msws2016.rst:87]:
```suggestion
  - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``Name ID``.
```

File: source/deployment/sso-saml-adfs.rst [hunk 4]
```diff
@@ -85,11 +85,12 @@ Create Claim Rules
 
 3. In the **Configure Claim Rule** screen, enter a **Claim Rule Name** of your choice, select **Active Directory** as the **Attribute Store** and do the following:
   - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``Email``.
+  - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``NameID``.
```
Comment (sbishel) [inline source/deployment/sso-saml-adfs.rst:88]:
```suggestion
  - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``Name ID``.
```

**Review by sbishel — CHANGES_REQUESTED** [review]

**Review by sbishel — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**